File: app/src/main/java/br/com/jonathanzanella/myexpenses/source/Source.java
package br.com.jonathanzanella.myexpenses.source;

import android.support.annotation.NonNull;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import br.com.jonathanzanella.myexpenses.sync.UnsyncModel;

public class Source implements UnsyncModel {
	private long id;
	@Expose
	private String name;
	@Expose
	private String uuid;
	@Expose @SerializedName("_id")
	private String serverId;
	@Expose @SerializedName("created_at")
	private long createdAt;
	@Expose @SerializedName("updated_at")
	private long updatedAt;
	private boolean sync;

	@NonNull
	@Override
	public String getData() {
		return "name=" + name +
				"\nuuid=" + uuid +
				"\nserverId=" + serverId;
	}

	@Override
	public long getId() {
		return id;
	}

	@Override
	public void setId(long id) {
		this.id = id;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	@Override
	public String getUuid() {
		return uuid;
	}

	public void setUuid(String uuid) {
		this.uuid = uuid;
	}

	@Override
	public String getServerId() {
		return serverId;
	}

	@Override
	public void setServerId(String serverId) {
		this.serverId = serverId;
	}

	@Override
	public long getCreatedAt() {
		return createdAt;
	}

	@Override
	public void setCreatedAt(long createdAt) {
		this.createdAt = createdAt;
	}

	@Override
	public long getUpdatedAt() {
		return updatedAt;
	}

	@Override
	public void setUpdatedAt(long updatedAt) {
		this.updatedAt = updatedAt;
	}

	@Override
	public boolean getSync() {
		return sync;
	}

	@Override
	public void setSync(boolean sync) {
		this.sync = sync;
	}
}

File: app/src/main/java/br/com/jonathanzanella/myexpenses/source/Source.kt
package br.com.jonathanzanella.myexpenses.source

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

import br.com.jonathanzanella.myexpenses.sync.UnsyncModel

class Source : UnsyncModel {
    override var id: Long = 0
    @Expose
    var name: String? = null
    @Expose
    override var uuid: String? = null
    @Expose
    @SerializedName("_id")
    override var serverId: String? = null
    @Expose
    @SerializedName("created_at")
    override var createdAt: Long = 0
    @Expose
    @SerializedName("updated_at")
    override var updatedAt: Long = 0
    override var sync: Boolean = false

    override fun getData(): String {
        return "name=" + name +
                "\nuuid=" + uuid +
                "\nserverId=" + serverId
    }
}